packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -0,0 +1,313 @@
+                <div className='account-types__box-icons'>
+                    {icons.map((icon, index) => {
+                        return <Icon className='account-types__box-icon' icon={icon} key={index} />;
+                    })}
+                </div>
+            </div>
+            {cards}
+        </div>
+    );
+};
+
+Box.propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+    footer_text: PropTypes.string,
+    icons: PropTypes.arrayOf(PropTypes.string),
+    cards: PropTypes.array,
+};
+
+const FinancialBox = ({ is_demo = false }) => {
+    return (
+        <Box
+            title={localize('Financial account ({{type}})', { type: is_demo ? localize('Demo') : localize('Real') })}
+            description={localize(
+                'Financial accounts offer you to trade financial assets such as Forex, commodities and indices.'
+            )}
+            footer_text={localize('Over 50 tradable financial assets')}
+            icons={['IcUnderlyingFRXNZDJPY', 'IcUnderlyingOTC_NDX', 'IcUnderlyingFRXXAUUSD', 'IcUnderlyingFRXBROUSD']}
+            cards={[
+                <AccountCard
+                    key={0}
+                    title={localize('Trade on Deriv')}
+                    subtitle={localize('Option trading account')}
+                    button_text={is_demo ? localize('Add demo account') : localize('Add real account')}
+                    buttonOnClick={() => {}}
+                    items={[
+                        {
+                            label: localize('Multiplier'),
+                            value: localize('Up to X1000'),
+                        },
+                        {
+                            label: localize('Stop loss'),
+                            value: localize('Flexible'),
+                        },
+                        {
+                            label: localize('Take profit'),
+                            value: localize('Flexible'),
+                        },
+                        {
+                            label: localize('Cancel Trade'),
+                            value: localize('Allow'),
+                        },
+                        {
+                            label: localize('Currency'),
+                            value: localize('USD/GBP/EUR'),
+                        },
+                    ]}
+                    learn_more={[
+                        {
+                            text: localize('Option Trading'),
+                            path: '/0',
+                        },
+                        {
+                            text: localize('Dtrader'),
+                            path: '/1',
+                        },
+                        {
+                            text: localize('DBot'),
+                            path: '/2',
+                        },
+                    ]}
+                >
+                    <p className='account-card__description__text'>
```

Comment:
Can we change this to `account-card__description`?, or is there a better way to describe this class? How about `account-card__text`?